src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -592,7 +571,7 @@ function GlobeControls(view, target, radius, options = {}) {
         }
         // MOVE_GLOBE
         // Rotate globe with mouse
-        if (state === CONTROL_STATE.MOVE_GLOBE) {
+        if (state === this.cfgControl.MOVE_GLOBE.state) {
```

Comment:
`cfgControl` -> `configuration` or `config`

Reply:
What about removing `MOVE_GLOBE.state` and directly use `state = this.cfgControl.MOVE_GLOBE` instead?

Reply:
Or `this.states` instead of `this.cfgControl` ?